src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example1.java

```diff
@@ -0,0 +1,26 @@
+/*
+SeparatorWrap
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.separatorwrap;
+// xdoc section -- start
+import java.io.
+          IOException; // OK, dot is on the previous line
+
+class Example1 {
+  String s;
+
+  public void foo(int a,
+                    int b) { // OK, comma is on the previous line
+  }
+
+  public void bar(int p
+                    , int q) { // violation '',' should be on the previous line'
+    if (s
+          .isEmpty()) { // violation ''.' should be on the previous line'
+    }
+  }
+}
+// xdoc section -- end
```

Comment:
We need check for new line at end of example files

Reply:
Issue created at https://github.com/checkstyle/checkstyle/issues/13244

Reply:
Done. File is fixed.